releasenotes/v0.10.0.md

```diff
@@ -0,0 +1,115 @@
+# Changes since v0.9.0
+
+## :warning: Breaking Changes
+
+- Bump Kubernetes api, controller-runtime and decouple webhooks from api (#2390)
+- Only accept HTTPHeadersRef in same namespace [GHSA-c98h-7hp9-v9hq](https://github.com/metal3-io/baremetal-operator/security/advisories/GHSA-c98h-7hp9-v9hq)
+
+## :sparkles: New Features
+
+- Use standard GetVirtualMedia API for checking dataImage status (#2336)
+- Add support for Disablepoweroff (#2229)
+- add support for locally built ironic in e2e test (#2167)
+
+## :bug: Bug Fixes
+
+- Fix few errors when making beta release notes (#2381)
+- Handle dataImage before reboot annotation to prevent annotation removal before all actions are complete (#2333)
+- fix broken release process (#2325)
+- Define some IPA (downloader) related variables (#2264)
+- bump x/net, x/crypto and x/oauth2 (#2316)
+- update rescue iso used for e2e tests (#2315)
+- Add BMH OwnerReference for DataImage right after its creation (#2249)
+- bump golang to 1.23.6 (#2243)
+- Set ownerReference for dataImage during BMH Reconcile (#2238)
+- Do not create HFC for non-redfish BMH (#2227)
+- bump golang to 1.23.5 (#2224)
+- Set e2e timeout to 1.5h (#2191)
+- Add validation against cross-namespace secret references (#2190)
+- Adding empty checksumType checks on BMH  to automatically detect hashing algorithm (#2174)
+- Fix BMO optional test by not checking metrics service (#2170)
+- Fix metrics service validation (#2165)
+
+## :seedling: Others
+
+- Bump CAPI to v1.10.0 (#2416)
+- Bump github.com/prometheus/client_golang from 1.21.1 to 1.22.0 (#2411)
+- Bump golang.org/x/crypto from 0.36.0 to 0.37.0 in /hack/tools (#2412)
+- Bump golang.org/x/crypto from 0.36.0 to 0.37.0 in /test (#2414)
+- Bump github.com/onsi/ginkgo/v2 from 2.23.3 to 2.23.4 in /test (#2413)
+- Refactoring: move ./controllers to ./internal/controller (#2405)
+- Makefile: do not explicit include ./internal/ (#2404)
+- Add Adil and Peppi-lotta as reviewers (#2407)
+- Bump the kubernetes group across 2 directories with 1 update (#2400)
+- Bump go.etcd.io/etcd/client/pkg/v3 from 3.5.19 to 3.5.21 (#2367)
+- Minor fix release workflow (#2403)
+- Enable intrange linter in golangci-lint (#2393)
+- Bump github.com/onsi/gomega from 1.36.3 to 1.37.0 in /test (#2388)
+- make osv-scanner scan run daily (#2389)
+- add labels to Dockerfile (#2384)
+- Enable testifylint linter in golangci-lint (#2383)
+- Bump golang.org/x/oauth2 from 0.27.0 to 0.29.0 in /hack/tools (#2387)
+- Bump github.com/onsi/gomega from 1.36.3 to 1.37.0 (#2386)
+- bump golang to v1.23.8 and x/net to v0.38.0 (#2374)
+- Enable usetesting linter in golangci-lint (#2380)
+- cleanup build/ and obsolete publishing-images.md (#2379)
+- Bump the github-actions group with 5 updates (#2378)
+- do not bump major version of github actions (#2373)
+- Minor logging improvements (#2370)
+- Bump sigs.k8s.io/kind from 0.26.0 to 0.27.0 in /test (#2327)
+- Remove keepalived from BMO repository (#2228)
```

Comment:
This probably should be in breaking or somehow otherwise highlighted here. We practically stop tagging it in release process with BMO tags, so we probably break few things (manifests, scripts etc) by not tagging it.

Reply:
done